package.json

```diff
@@ -17,6 +17,7 @@
     "prepare": "husky",
     "build": "npx hardhat compile",
     "test": "npx hardhat test",
+    "test-ci": "npm run test && FACTORY=false npm run test",
```

Comment:
Target is to run tests with contracts deployed through factory **and** through EOA (at least for now)